### ISSUE TITLE
TODO list

### ISSUE THREAD

**Issue by yetanotherx** [pdaian/MCBouncer#4]
Guide:
- <strong>Need to be finished before release</strong>
- <del>Completed items</del>
- <em>Can wait until after release</em>

---
- Backups w/ restore
- <em>Unit tests</em>
- Verify that global notes are seen as such
- Documentation
- Find some better homepage icons
- <em>Timeline</em>
- <em>Admin interface</em>
- Forum
- Imports
- Disable server
- <del>Numeric usernames</del>
- <13 year olds in privacy policy

**Comment by isiah-lloyd** [comment]
Can you also add it so there is a permission to see who got banned?

**Comment by Deaygo** [comment]
Do you mean to see the announcements for when people are banned?

**Comment by isiah-lloyd** [comment]
Yes! Also this is more of a site thing but can we have it so we can see the total bans on our server. and a export feature?
